### PR TITLE
Added PSU CLI

### PR DESCRIPTION
**Pull request by andrii-savka**

### CODE REVIEW

File: psuutil/main.py [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+    import traceback
+    from tabulate import tabulate
+except ImportError as e:
+    raise ImportError("%s - required module not found" % str(e))
+
+VERSION = '1.0'
+
+SYSLOG_IDENTIFIER = "psuutil"
+PLATFORM_SPECIFIC_MODULE_NAME = "psuutil"
+PLATFORM_SPECIFIC_CLASS_NAME = "PSUutil"
```
Comment (jleveque) [inline psuutil/main.py]: Suggest changing case of class name to `PsuUtil` to match CamelCase of `SfpUtil`

File: psuutil/main.py [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+# 'version' subcommand
+@cli.command()
+def version():
+    """Display version info"""
+    click.echo("psuutil version {0}".format(VERSION))
+
+# 'numpsus' subcommand
+@cli.command()
+def numpsus():
+    "number of supported PSU in the device"
```
Comment (jleveque) [inline psuutil/main.py]: Suggest `"Display number of supported PSU in the device"`

File: sonic_psu/psu_base.py [hunk 3]
```diff
@@ -26,10 +26,22 @@ def get_num_psus(self):
     @abc.abstractmethod
     def get_psu_status(self, index):
         """
-        Retrieves the oprational status of power supply unit (PSU) defined
+        Retrieves the operational status of power supply unit (PSU) defined
                 by index <index>
 
         :param index: An integer, index of the PSU of which to query status
         :return: Boolean, True if PSU is operating properly, False if PSU is faulty
```
Comment (jleveque) [inline sonic_psu/psu_base.py]: Could you also please add more detail to my description here to reflect what we have discussed as "faulty" (e.g., if the PSU is present but powered off, it is considered a fault, and should return false.

File: psuutil/main.py (hunk 2 above)
Comment (andrii-savka) [reply psuutil/main.py]: Done

File: sonic_psu/psu_base.py (hunk 3 above)
Comment (andrii-savka) [reply sonic_psu/psu_base.py]: More details has been added

File: sonic_psu/psu_base.py [hunk 4]
```diff
@@ -26,10 +26,22 @@ def get_num_psus(self):
     @abc.abstractmethod
     def get_psu_status(self, index):
         """
-        Retrieves the oprational status of power supply unit (PSU) defined
+        Retrieves the operational status of power supply unit (PSU) defined
                 by index <index>
 
         :param index: An integer, index of the PSU of which to query status
```
Comment (yurypm) [inline sonic_psu/psu_base.py]: Is it zero-based index? I think this info should be here.

Comment (andrii-savka) [reply sonic_psu/psu_base.py]: Added more details

**Pull request merged**